packages/runtime/src/schema.json

```diff
@@ -5,40 +5,68 @@
   },
```

Comment:
why is this changed?

Reply:
What exactly are you pointing at ? if its link_prefixes its due to navigation when I tried to fix redirect issue, its not give any diff tho , i'll revert it back

Reply:
And I added some new compoundComponent just wanna try out @arrizalamin